### ISSUE TITLE
Keyboard layout on client is not correctly applied

### ISSUE THREAD

**Issue by arhimondr** [symless/synergy-core#6401]
### Operating Systems ###

Server: Ubuntu 18.04.1 LTS
Client: macOS High Sierra 10.13.6

### Synergy Version ###

- Synergy 1: `synergys 1.9.1, protocol version 1.6`
- Synergy 2: `1.9.1-stable-2a0225c1`

### Steps to Reproduce ###

1. On client switch layout to "Russian"
2. Type `ю` (`.`). Expected: `Ю` is the input, actual: `.`
3. Type `б` (`,`). Expected: `б` is the input, actual: `,`
4. Type `х` (`[`). Expected: `х` is the input, actual: `[`
5. Type `ъ` (`]`). Expected: `ъ` is the input, actual: `]`
6. Type `ж` (`;`). Expected: `ж` is the input, actual: `;`
7. Type `э` (`'`). Expected: `э` is the input, actual: `'`

In other words the layout is not being applied for this keys. Also the layout is not applied for `SHIFT+2,3,4,5,6,7,8`

Similar issue applies for `Ukrainian-PC` layout as well.

### Run Method ###

Config app

### Log URLs ###

Server: https://pastebin.com/SjwHpcCf
Client: https://pastebin.com/naGSDbkk

**Comment by arhimondr** [comment]
I didn't try the latest version. But based on the release logs it doesn't seem this issue could've been fixed.

**Comment by nbolton** [comment]
> I created an issue via the support form like a 2 months ago, but no one has replied to me yet.

@arhimondr Sorry to hear that! We normally reply to all tech support tickets within 24 hours, did you get a reply in the end? If not, perhaps your message failed to send or was not received.

**Comment by arhimondr** [comment]
@nbolton Nobody has ever replied to me =\

**Comment by acostroma** [comment]
The same issue on version 1.13 stable.
Any chance/ETA to get it fixed?
